frontend/src/components/AgencyProfile/AgencyProfile.css

```diff
@@ -287,95 +287,3 @@
 .rescue-wrapper p {
   margin-bottom: 0.5em;
 }
-
-/* * * * Tasks section * * * */
```

Comment:
I'm not sure if this was removed on purpose? If it was then it's fine, but I just wanted to double check

Reply:
Yeah, those CSS classes were moved to a different file and got re-added by accident (we love git merges). Thanks for checking in though!!